data/sql/other_transformation_custom.sql

```diff
@@ -1,5 +1,21 @@
 -- This file is hand generated.
 
+-- Norway triangulated files
+ 
+INSERT INTO other_transformation VALUES(
+    'PROJ','NGO48_TO_ETRS89NO','NGO 1948 to ETRS89 (2)',
+    'Transformation based on a triangulated irregular network',
+    'PROJ','PROJString',
+    '+proj=pipeline ' ||
+        '+step +proj=axisswap +order=2,1 ' ||
+        '+step +proj=tinshift +file=no_kv_ETRS89NO_NGO48_TIN.json +inv ' ||
```

Comment:
the file shoul also be registered in grid_alternatives.sql similarly to the Finnish ones